sormas-backend/src/main/java/de/symeda/sormas/backend/deletionconfiguration/AbstractCoreEntityFacade.java

```diff
@@ -55,4 +70,16 @@ protected String getDeleteReferenceField(DeletionReference deletionReference) {
 			throw new IllegalArgumentException("deletion reference " + deletionReference + " not supported in " + getClass().getSimpleName());
 		}
 	}
+
+	private Object[] getDeletionData(String uuid, DeletionConfiguration entityConfig) {
```

Comment:
Please add null checks in this method as well; `getDeleteReferenceField` will throw an exception when entityConfig.deletionReference is null. That should be prevented (i.e. instead this method should return null as well).